onnxruntime/core/graph/contrib_ops/bert_defs.cc

```diff
@@ -317,6 +318,12 @@ ONNX_MS_OPERATOR_SET_SCHEMA(
         .Attr("num_heads", "Number of attention heads", AttributeProto::INT)
         .Attr("mask_filter_value", "The value to be filled in the attention mask. Default value is -10000.0f",
               AttributeProto::FLOAT, OPTIONAL_VALUE)
+        .Attr("scale",
+              "Custom scale will be used if specified. Default value is 1/sqrt(head_size)",
+              AttributeProto::FLOAT,
+              OPTIONAL_VALUE)
+        .Attr("static_kv", "Whether to use static key and value(Cross-Attention). Default value is 1.",
```

Comment:
Why do we need this attribute?

If an attention has static key van value, it means key and value does not change (or no needed for past state). But from interface of this operator, it does not matter, and you can remove this attribute and the operator still works well.